docs/man/docker-ps.1.md

```diff
@@ -1,6 +1,6 @@
 % DOCKER(1) Docker User Manuals
 % Docker Community
-% JUNE 2014
+% FEBRURAY 2015
```

Comment:
lol, I read twice and didn't notice that. I thought you wanted for me to change the CAPS.